content/further-education-teacher-training.md

```diff
@@ -1,5 +1,10 @@
 ---
 title: "Further education teacher training"
+description: |-
+  Teaching in further education means teaching a wide range of subjects
```

Comment:
Just thinking how we might make this more succinct and avoid repetition of 'teaching'. 

Maybe: "You can teach a wide variety of subjects as a further education teacher. You'll inspire students aged 14 and above to build their skills and knowledge."

Reply:
How about "Being a further education teacher will give you the opportunity to teach a wider range of subjects"?

Reply:
this works ^